### PR TITLE
SEO edits Teams-quick-start-edu

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited metadata title per SEO recommendation. Edited description to add keywords and lengthen. Added a sentence to the third paragraph to include requested keywords. The examples I used are based on what I know of Teams capabilities, but should be verified for accuracy. Edited one bullet point to include a keyword.

@DaniHalfin

### CODE REVIEW

**Comment by DaniHalfin** [comment]
Reviewing. Thanks.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Teams-quick-start-edu.yml [hunk 1]
```diff
@@ -31,10 +31,10 @@ items:
 
     With Microsoft Teams, your school or institution can create collaborative classrooms, connect in professional learning communities, communicate with school staff, coordinate research across institutions, or more easily facilitate student life efforts like clubs or extracurricular activities – all from a single experience in Office 365 for Education.
 
-    Microsoft Teams saves time, deepens connections and simplifies everyday logistics. This allows staff to focus on their roles as educator, researchers, and leaders in your school or institution.  
+    Microsoft Teams saves time, deepens connections and simplifies everyday logistics. This allows staff to focus on their roles as educators, researchers, and leaders in your school or institution. Students benefit from easy-to-access online classrooms, recorded lessons, and one-on-one meetings with teachers.
```
Comment (JohanFreelancer9) [inline Teams/Teams-quick-start-edu.yml:34]:
```suggestion
    Microsoft Teams saves time, deepens connections, and simplifies everyday logistics. This allows staff to focus on their roles as educators, researchers, and leaders in your school or institution. Students benefit from easy-to-access online classrooms, recorded lessons, and one-on-one meetings with teachers.
```

**Comment by scanum** [comment]
@DaniHalfin Hi! Is this PR going to be closed? Thanks!

**Pull request merged**